### PR TITLE
NF: remove warning if can be simplified

### PR DESCRIPTION
**Pull request by Arthur-Milchior**

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldController.java [hunk 1]
```diff
@@ -203,11 +203,7 @@ private boolean canUseCamera(Context context) {
         }
 
         // Some hardware has no camera or reports yes but has zero (e.g., cheap devices, and Chromebook emulator)
-        if (CompatHelper.getCompat().getCameraCount() < 1) {
-            return false;
-        }
-
-        return true;
+        return CompatHelper.getCompat().getCameraCount() >= 1;
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldController.java:206]: (nit) I prefer the previous style when checking a list of boolean conditions. Should I stop doing this.

Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldController.java:206]: I’m fine adding a suppress warning sign. I guess a consistant list is indeed nice

Comment (david-allison-1) [reply AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldController.java:206]: Seems more sensible for me to change my preference if nobody else feels strongly and it'll make linting easier

**Review by david-allison-1 — APPROVED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldController.java (hunk 1 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldController.java]: I just checked the whole method and I agree with you. So I just deleted the warning here.

Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/anki/multimediacard/fields/BasicImageFieldController.java]: I just checked the whole method and I agree with you. So I just deleted the warning here.

**Pull request merged**